src/repo.ts

```diff
@@ -88,23 +174,90 @@ export async function getRepositoryHttpsUrl(
  * of the new release).
  * 3. Switches to that branch.
  *
- * @param projectRepositoryPath - The path to the repository directory.
+ * @param repositoryDirectoryPath - The path to the repository directory.
  * @param args - The arguments.
  * @param args.releaseVersion - The release version.
  */
 export async function captureChangesInReleaseBranch(
-  projectRepositoryPath: string,
+  repositoryDirectoryPath: string,
   { releaseVersion }: { releaseVersion: string },
 ) {
-  await getStdoutFromGitCommandWithin(projectRepositoryPath, [
-    'checkout',
+  await getStdoutFromGitCommandWithin(repositoryDirectoryPath, 'checkout', [
     '-b',
     `release/${releaseVersion}`,
   ]);
-  await getStdoutFromGitCommandWithin(projectRepositoryPath, ['add', '-A']);
-  await getStdoutFromGitCommandWithin(projectRepositoryPath, [
-    'commit',
+  await getStdoutFromGitCommandWithin(repositoryDirectoryPath, 'add', ['-A']);
+  await getStdoutFromGitCommandWithin(repositoryDirectoryPath, 'commit', [
     '-m',
     `Release ${releaseVersion}`,
   ]);
 }
+
+/**
+ * Retrieves the names of the tags in the given repo, sorted by earliest created
+ * to latest created. As this fetches tags from the remote first, you are
```

Comment:
Nit: Update description to match change in sort order

```suggestion
 * Retrieves the names of the tags in the given repo, sorted by tag name in
 * ascending order. As this fetches tags from the remote first, you are
```

Reply:
Good catch. Maybe for clarity we say "ascending semantic version order"?

Reply:
Sounds reasonable!

Reply:
Fixed in [8190f5f](https://github.com/MetaMask/create-release-branch/pull/23/commits/8190f5f0a7cc46ec85269ed8645c909df555e757).